tests/lib/mock_nm.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3 -tt
```

Comment:
I like the test helper module.
But what are these two first lines?
And we need to have the licensing statement here.

Reply:
Good catch, I fixed this.